src/views/index/index.vue

```diff
@@ -20,8 +20,12 @@
               Migrate your <span class="ae-brand">AE tokens</span> to the Mainnet
             </h1>
             <p class="text text-intro text-intro_light">
-              The æternity mainnet is almost here! You can begin migrating the AE tokens you own on Ethereum to the æternity mainnet! We will guide you through the secure migration process.
+              The æternity mainnet is here! The AE tokens you now own on Ethereum can soon be migrated to the æternity mainnet! Here we will guide you through the secure migration process.
```

Comment:
I think we should rather say that the Phase0 is over. End of phase does not mean that mainnet is here. We need some time to generate the genesis block and acually launch after the end of the phase.

Reply:
I agree. Let's write "Phase 0 is over" instead of "The æternity mainnet is here!"

Reply:
agreed, this has now been changed